app/experimenter/experiments/forms.py

```diff
@@ -860,19 +864,41 @@ def required_reviews(self):
             self["review_bugzilla"],
             self["review_qa"],
             self["review_relman"],
-        )
+        ]
+        return self.add_conditional_reviews(default_required)
+
+    def add_conditional_reviews(self, required_reviews):
+        for review, risk in self._required_reviews_mapping.items():
+            if risk:
+                self._remove_optional_label(review)
+                self._change_help_text_to_risk_help(review)
+                required_reviews.append(self[review])
+        return required_reviews
+
+    def _remove_optional_label(self, review):
+        self[review].label = self[review].label.strip("(Optional)")
+
+    def _change_help_text_to_risk_help(self, review):
+        self[review].help_text = Experiment.RISK_GENERAL_HELP_TEXT
```

Comment:
I'm not sure I completely understand this. My reading of the linked issue is that the link you have here is for the `Is this partner related?` question, but that the review should instead link to https://mana.mozilla.org/wiki/display/FIREFOX/Pref-Flip+and+Add-On+Experiments#Pref-FlipandAdd-OnExperiments-DependentSign-offs instead.

Reply:
Thinking about this more, especially since the linked issue requires the text of the checklist item to change, it may not be possible to do this in a generic way (the way you have).

Reply:
my bads, I definitely read the issue wrong